tests/test_contracts/test_class_contracts.py

```diff
@@ -25,7 +26,7 @@ class Person:
 
     age: int
     name: str
-    fav_foods: List[str]
+    fav_foods: list[str]
```

Comment:
Okay so the tests on 3.8 are failing, and not just because of the flaky test we previously discussed. I realized that because 3.8 doesn't support the newer type annotations, we can't change these type annotations *for functions and classes that are used for contract-checking testing. This includes the functions/classes in this file and other files under `tests/test_contracts`. Please revert those changes.